src/fstests/fstests.jinja2

```diff
@@ -135,30 +142,24 @@ class Job(BaseJob):
         return node
 
     def _run(self, src_path):
-        fail_results = {
-            'node': {
-                'name': 'fstests',
-                'result': 'fail',
-                'state': 'done',
-            },
-            'child_nodes':[],
-        }
-        if not self._kernel_config(src_path):
-            return fail_results
-        if not self._kernel_build(src_path):
-            return fail_results
-        if not self._run_smoke_tests():
-            return fail_results
-        if not self._get_xml_from_vm(src_path):
-            return fail_results
+        results = self._generate_node_result('fstests')
+        ok, result_node = self._kernel_config(src_path)
```

Comment:
`ok` doesn't seem to be a proper variable name to me.

Reply:
`ok` is to confirm if the `config` and `build` steps were ok! But to improve the readability, I changed to `config/build_success`

Reply:
It shows status rather than success.
How about `config_status` and `build_status`?

Reply:
@JenySadadia To be honest, I am not sure about this; we have `True/False` and we need to know if config and build were run as expected (success as True or False). But, updated it with your suggestion.